frontend/src/views/CellGuide/components/CellGuideCard/components/MarkerGeneTables/hooks/canonical_markers.ts

```diff
@@ -71,17 +76,15 @@ function _getReferenceData(
 function _passSelectionCriteria({
   markerGene,
   allTissuesLabelToIdMap,
-  selectedOrganLabel,
   selectedOrganId,
 }: {
   markerGene: CanonicalMarkersQueryResponseEntry;
   allTissuesLabelToIdMap: Map<string, string>;
-  selectedOrganLabel: string;
   selectedOrganId: string;
 }): boolean {
   // There are marker genes tissues labeled as "All Tissues"
   // so select them only when selectedOrganLabel is "All Tissues"
-  if (selectedOrganLabel === ALL_TISSUES) {
+  if (selectedOrganId === NO_ORGAN_ID) {
```

Comment:
i'm using selected organ id instead of selected organ label because i think it's a bit better to use the id (which represents the actual entity) instead of a label (which is just for view purposes)